core/proto/core.proto

```diff
@@ -12,6 +12,7 @@ message Challenge {
   string keyAuthorization = 5;
   repeated ValidationRecord validationrecords = 10;
   ProblemDetails error = 7;
+  string validated = 11;
```

Comment:
We use `int64` for the Authorization expiration time, so I'd prefer to do that here as well. I'd argue that it is better to special-case `0` than it is to use a string to represent something that we represent as an int everywhere else.